lms/djangoapps/discussion/tests/test_signals.py

```diff
@@ -19,62 +18,51 @@ def setUp(self):
 
     @mock.patch('lms.djangoapps.discussion.signals.handlers.get_current_site')
     @mock.patch('lms.djangoapps.discussion.signals.handlers.send_message')
-    @override_waffle_flag(SEND_NOTIFICATIONS_FOR_COURSE, True)
     def test_comment_created_signal_sends_message(self, mock_send_message, mock_get_current_site):
         site_config = SiteConfigurationFactory.create(site=self.site)
         site_config.values[ENABLE_FORUM_NOTIFICATIONS_FOR_SITE_KEY] = True
         site_config.save()
         mock_get_current_site.return_value = self.site
-        with waffle().override(FORUM_RESPONSE_NOTIFICATIONS):
-            signals.comment_created.send(sender=self.sender, user=self.user, post=self.post)
+        signals.comment_created.send(sender=self.sender, user=self.user, post=self.post)
 
-            mock_send_message.assert_called_once_with(self.post, mock_get_current_site.return_value)
+        mock_send_message.assert_called_once_with(self.post, mock_get_current_site.return_value)
 
     @mock.patch('lms.djangoapps.discussion.signals.handlers.send_message')
-    @override_waffle_flag(SEND_NOTIFICATIONS_FOR_COURSE, True)
     def test_comment_created_signal_message_not_sent_without_waffle_switch(self, mock_send_message):
```

Comment:
OK, so now you can actually get rid of this test - we only want to keep the tests related to site configuration.